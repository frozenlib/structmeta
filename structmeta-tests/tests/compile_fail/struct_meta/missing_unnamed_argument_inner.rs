#[structmeta_tests::attr_required_unnamed_param2_inner(value(10), after = 20)]
fn my_func() {}

fn main() {}
