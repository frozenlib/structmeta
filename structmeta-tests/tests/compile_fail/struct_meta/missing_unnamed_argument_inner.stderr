error: expected least 2 arguments but 1 argument was supplied
 --> $DIR/missing_unnamed_argument_inner.rs:1:61
  |
1 | #[structmeta_tests::attr_required_unnamed_param2_inner(value(10), after = 20)]
  |                                                             ^^^^
